internal/image/ostree_installer.go

```diff
@@ -114,7 +114,7 @@ func (img *OSTreeInstaller) InstantiateManifest(m *manifest.Manifest,
 
 	isoPipeline := manifest.NewISO(m, buildPipeline, isoTreePipeline)
 	isoPipeline.Filename = img.Filename
-	isoPipeline.ISOLinux = true
+	isoPipeline.ISOLinux = img.Platform.GetArch() == platform.ARCH_X86_64
```

Comment:
Potentially for follow-up (or just to consider): should the platform interface have a `ISOLinuxSupported()` function that we would query instead of this logic here?

Reply:
I don't think that's worth the effort. isolinux is x86 only, and it is going away in favor of booting everything using grub2.

Reply:
I considered making it more explicit like you mentioned Tom, but Brian's point is valid.  Maybe we could do it for now just for the _cleanliness_.

Reply:
Yup, agreed.